static_src/components/users_invite.jsx

```diff
@@ -7,7 +7,6 @@
 import React from 'react';
 
 import Action from './action.jsx';
-import ErrorMessage from './error_message.jsx';
```

Comment:
when does it get called? i didn't see an error for it in the build before it was fixed.